Branch-SDK/Branch-SDK/BranchGetAppListRequest.h

```diff
@@ -1,16 +0,0 @@
-//
```

Comment:
Ah, i was talking about the removal of the two classes (.h & .m files) that we removed, caused crashes, and i added back last week. Just want to make sure it's safe to finally remove them now?